examples/Makefile

```diff
@@ -1,11 +1,12 @@
-PROJECT_NAME=resources
+PROJECT_NAME=examples
 
 RELEASE_VERSION ?= latest
 RELEASE_PATH ?= ../strimzi-$(RELEASE_VERSION)/$(PROJECT_NAME)
 
 release:
 	mkdir -p $(RELEASE_PATH)
+	cp -r ./common $(RELEASE_PATH)/
```

Comment:
Are these path correct? In `example/` are currently directories `openshift`, `resources` and `install`. Shouldn't these be copied?

Reply:
Good catch! I should've tested the `release` target. If I understood `make` better it would make this testing easier.